flink-tests/src/test/java/org/apache/flink/test/runtime/minicluster/LocalFlinkMiniClusterITCase.java

```diff
@@ -49,7 +49,7 @@
  */
 public class LocalFlinkMiniClusterITCase extends TestLogger {
 
-	private static final String[] ALLOWED_THREAD_PREFIXES = { };
+	private static final String[] ALLOWED_THREAD_PREFIXES = { "initialSeedUniquifierGenerator" };
```

Comment:
Please format it as follows:
```
private static final String[] ALLOWED_THREAD_PREFIXES = { 
    // ENTER_YOUR_COMMENT_HERE
    "initialSeedUniquifierGenerator" 
};
```
and add some comment describing what is an issue with this thread (that it is background/daemon thread and it sometimes is spawned before this test and sometimes it is spawned during this test)